mitmproxy/utils/typecheck.py

```diff
@@ -1,20 +1,37 @@
 import typing
 
 
-def check_type(name: str, value: typing.Any, typeinfo: typing.Any) -> None:
+def check_command_return_type(value: typing.Any, typeinfo: typing.Any) -> bool:
```

Comment:
I don't get this distinction. My understanding so far was that these functions are not really be specific to options or commands. If you want to check the return type, you can just `check_type("the return value", val, type)` ? 
I'd also prefer raising over returning here so that we have a more informative error message if required.